tests/plugins/test_rpmqa.py

```diff
@@ -221,33 +235,57 @@ def test_rpmqa_plugin_base_from_scratch(caplog, tmpdir, docker_tasker, packages_
         assert workflow.image_components is None
 
     else:
-        if not packages_exists:
-            results = runner.run()
-            log_msg = '{} does not exist in rpmdb'.format(RPMDB_PACKAGES_NAME)
-            assert log_msg in caplog.text
-            assert results[PostBuildRPMqaPlugin.key] is None
+        log_msg_getting = 'getting rpms from rpmdb:'
+        if rpm_failed:
+            with pytest.raises(Exception, match='rpm query failed'):
+                runner.run()
+            assert log_msg_getting in caplog.text
             assert workflow.image_components is None
+            log_msg = 'Failed to get rpms from rpmdb:'
+            assert log_msg in caplog.text
 
         else:
-            log_msg_getting = 'getting rpms from rpmdb:'
-            if rpm_failed:
-                with pytest.raises(Exception, match='rpm query failed'):
-                    runner.run()
-                assert log_msg_getting in caplog.text
-                assert workflow.image_components is None
-                log_msg = 'Failed to get rpms from rpmdb:'
-                assert log_msg in caplog.text
-
-            else:
-                results = runner.run()
-                assert log_msg_getting in caplog.text
-                assert results[PostBuildRPMqaPlugin.key] == PACKAGE_LIST
-                assert workflow.image_components == parse_rpm_output(PACKAGE_LIST)
+            results = runner.run()
+            assert log_msg_getting in caplog.text
+            assert results[PostBuildRPMqaPlugin.key] == PACKAGE_LIST
+            assert workflow.image_components == parse_rpm_output(PACKAGE_LIST)
 
     log_msg = "from scratch, list_rpms_from_scratch is True, trying get rpmdb"
     assert log_msg in caplog.text
 
 
+@pytest.mark.parametrize('rpmdb_dir_exists, rpmdb_dir_empty', [(True, False), (False, False)])
```

Comment:
merge this test with previous test, which was already testing similar thing before